packages/yourloops/pages/hcp/profile-dialog.tsx

```diff
@@ -0,0 +1,150 @@
+ */
+
+import { Units, User } from "../../models/shoreline";
+import React, { FunctionComponent } from "react";
+import {
+  Button,
+  createStyles,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  makeStyles,
+  TextField,
+  Theme,
+} from "@material-ui/core";
+import { useTranslation } from "react-i18next";
+
+interface ProfileDialogProps {
+  user: User;
+  isOpen: boolean;
+  handleClose: () => void;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    title: {
+      textAlign: "center",
+      color: theme.palette.primary.main,
+    },
+    textField: {
+      marginTop: "1em",
+      "& input:disabled": {
+        backgroundColor: "white",
+        color: theme.palette.grey[800],
+      },
+    },
+    disabled: {
+      "&&:before": {
+        borderBottom: "0.5px solid",
+        color: theme.palette.grey[400],
+      },
+    },
+  })
+);
+
+const ProfileDialog: FunctionComponent<ProfileDialogProps> = ({ user, isOpen, handleClose }: ProfileDialogProps) => {
+  const { t } = useTranslation("yourloops");
+  const { textField, title, disabled } = useStyles();
+
+  const mail = user?.emails ? user.emails[0] : "";
+  const hbA1c = "8.5%"; // TODO
+  const birthDate = user?.profile?.patient?.birthday?.split("T")[0];
+
+  return (
+    <Dialog fullWidth={true} maxWidth="xs" open={isOpen} onClose={handleClose}>
+      <DialogTitle className={title} id="patient-dialog-title">
+        {t("patient-profile")}
+      </DialogTitle>
+      <DialogContent>
+        <TextField
+          fullWidth
+          disabled
+          id="firstname"
+          label={t("firstName")}
+          value={user.profile?.firstName}
+          className={textField}
+          InputProps={{ classes: { disabled } }}
+        />
+        <TextField
+          fullWidth
+          disabled
+          id="lastname"
+          label={t("lastName")}
+          value={user.profile?.lastName}
+          className={textField}
+          InputProps={{ classes: { disabled } }}
+        />
+        <TextField
+          fullWidth
+          disabled
+          id="birthDate"
+          label={t("hcp-patient-profile-birthdate")}
+          value={birthDate}
+          className={textField}
+          InputProps={{ classes: { disabled } }}
+        />
+        <TextField
+          fullWidth
+          disabled
+          id="mail"
+          label={t("email")}
+          value={mail}
+          className={textField}
+          InputProps={{ classes: { disabled } }}
+        />
+        <TextField
+          fullWidth
+          disabled
+          id="hbA1c"
+          label={t("initial-hbA1c")}
+          value={hbA1c}
+          className={textField}
+          InputProps={{ classes: { disabled } }}
+        />
+        <TextField
+          fullWidth
+          disabled
+          id="units"
+          label={t("units")}
+          value={user.settings?.units?.bg || Units.gram}
```

Comment:
```suggestion
          value={user.settings?.units?.bg ?? Units.gram}
```
We can use this specific operator now